docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.3'
+services:
+  coordinator:
+    image: "labgrid-coordinator"
+    volumes:
+      - "./crossbar:/opt/.crossbar"
+    tty: true
+    network_mode: "host"
+    user: "${CURRENT_UID}"
+    command: bash -c "cp /opt/.crossbar/places_example.yaml /opt/.crossbar/places.yaml &&
+      crossbar start --config /opt/labgrid/.crossbar/config.yaml"
+  client:
+    image: "labgrid-client"
+    volumes:
+      - "./client/simple-test:/simple-test"
+      - "./client/.ssh:/root/.ssh"
+    tty: true
+    stdin_open: true
+    network_mode: "host"
+    tmpfs: "/tmp"
+    # Use wait-for-it to ensure exporter service is up, as exporter is assuming exporter to
+    # Use labgrid-client r to ensure the exporter has populated the resource list in the coordinator
+    command: bash -c "set -e &&
+      cd /simple-test &&
+      /opt/wait-for-it/wait-for-it.sh 127.0.0.1:20408 &&
+      while [ -z $$(/usr/local/bin/labgrid-client r) ]; do echo 'Wait one sec on coordinator' && sleep 1; done &&
+      /usr/local/bin/labgrid-client -p example-place unlock --kick ||
+      /usr/local/bin/labgrid-client -p example-place lock &&
+      /usr/local/bin/pytest --lg-env remote.yaml -s &&
+      /usr/local/bin/labgrid-client -p example-place unlock"
+    depends_on:
+      - coordinator
+      - exporter
+      - dut
+  exporter:
+    image: "labgrid-exporter"
+    volumes:
+      - "./exporter-conf:/opt/conf"
+    depends_on:
+      - coordinator
+    tty: true
+    network_mode: "host"
+    user: "${CURRENT_UID}"
+    stdin_open: true
+    command: bash -c "set -e &&
+      /opt/wait-for-it/wait-for-it.sh 127.0.0.1:20408 -- labgrid-exporter /opt/conf/exporter.yaml"
+  dut:
+    build:
+      context: "./dut"
+    network_mode: "host"
+    privileged: true
```

Comment:
Why does this need to be priviledged, especially if you have a well known root password?

Reply:
Actually the privileged keyword will ensure the container to be privileged in terms of what i can do on the host running the container hence it has nothing to do with the root user in the container. That being said you are right there is no good reason for the DUT to be privileged, hence I will remove it